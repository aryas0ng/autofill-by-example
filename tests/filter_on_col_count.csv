1,2
1,2
4,1
6,3
6,3
6,
19,
19,
19,
19,
9,
7,
7,
3,
3,
3,
1,
1,
1,
3,
3,
8,
8,