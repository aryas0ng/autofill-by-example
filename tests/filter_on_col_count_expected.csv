1,2
1,2
4,1
6,3
6,3
6,3
19,4
19,4
19,4
19,4
9,1
7,2
7,2
3,3
3,3
3,3
1,3
1,3
1,3
3,2
3,2
8,2
8,2